build.gradle

```diff
@@ -185,17 +192,26 @@ jar {
 
 //install4j tasks for building win/nix/mac installers. Executable jar can be built without it
 install4j {
-        installDir = file('/opt/install4j7')
-        //license must be passed via gradlew commandline -P as it can't be commit to public source control
-        if(project.hasProperty('gradlewCommandLicenseProp'))
-        {
-          license = gradlewCommandLicenseProp
-        }
+    //pass in the install4j directory via -P if it's not in the default install location, e.q. for use with travis or arbitrary paths
+    if(project.hasProperty('gradlewInstall4jDirectory'))
+    {
+    installDir = file(gradlewInstall4jDirectory)
```

Comment:
Nit: indentation

Reply:
Open to suggestions. I'm not good with that sort of stuff.

Reply:
done. Thanks.